aptos-move/framework/aptos-framework/sources/genesis.move

```diff
@@ -365,7 +365,7 @@ module aptos_framework::genesis {
             validator.consensus_pubkey,
             validator.proof_of_possession,
         );
-        stake::update_network_and_fullnode_addresses(
+        stake::force_update_network_and_fullnode_addresses(
```

Comment:
Why do we need this for genesis?

Reply:
some smoke tests set this flag (disable validator set change) which made genesis failed...